corehq/apps/reports/standard/forms/filters.py

```diff
@@ -21,15 +21,16 @@ class SubmissionErrorType(object):
     OVERWRITTEN = 3
     UNKNOWN_ERROR = 4
     ARCHIVED = 5
-    
+
+
     doc_types = ["XFormInstance", "XFormError", "XFormDuplicate", "XFormDeprecated", "SubmissionErrorLog", "XFormArchived"]
     human_readable = [ugettext_lazy("Normal Form"),
                       ugettext_lazy("Form with Errors"),
                       ugettext_lazy("Duplicate Form"),
                       ugettext_lazy("Overwritten Form"),
                       ugettext_lazy("Generic Error"),
                       ugettext_lazy("Archived Form")]
-    
+
     error_defaults = [False, True, False, False, True, False]
     success_defaults = [True, False, False, False, False, False]
```

Comment:
Tangential to this PR, but it looks like `success_defaults` isn't referenced anywhere and can be removed.

Reply:
Nevermind, I see now you took care of this.